resume_example.json

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
Do we want to check the example file in? If not we might want to add it to .gitignore

Reply:
I went ahead and deleted it; I don't think we need to check it in.